edb/lib/cal.edgeql

```diff
@@ -16,9 +16,7 @@
 # limitations under the License.
 #
 
-CREATE MODULE cal {
-    SET builtin := true;
-};
+CREATE MODULE cal;
```

Comment:
Why this change? Do `schema::Module` objects still have a `builtin` property? How to set it?

Reply:
*All* standard schema objects now have the property and have it set automatically at bootstrap.